code/addons/a11y/src/types.ts

```diff
@@ -1,3 +1,47 @@
-import type { AxeResults } from 'axe-core';
+import type { AxeResults, ElementContext, RunOptions, Spec } from 'axe-core';
 
 export type A11YReport = AxeResults | { error: Error };
+
+export interface A11yParameters {
+  /**
+   * Accessibility configuration
+   *
+   * @see https://storybook.js.org/docs/writing-tests/accessibility-testing
+   */
+  a11y?: {
+    /** Manual configuration for specific elements */
+    element?: ElementContext;
+
+    /**
+     * Configuration for the accessibility rules
+     *
+     * @see https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter
+     */
+    config?: Spec;
+
+    /**
+     * Options for the accessibility checks To learn more about the available options,
```

Comment:
I think the context makes it clear enough.

```suggestion
     * Options for the accessibility checks
```